src/passes/PostEmscripten.cpp

```diff
@@ -80,6 +80,9 @@ struct PostEmscripten : public Pass {
       runner->options.getArgumentOrDefault("emscripten-sbrk-ptr", "");
     if (sbrkPtrStr != "") {
       auto sbrkPtr = std::stoi(sbrkPtrStr);
+      if (sbrkPtr == 0 || sbrkPtr == -1) {
```

Comment:
How about just `sbrkPtr <= 0`?

Reply:
Hmm, in the future we should probably make this an unsigned value. Making it `<= 0` now might confuse things at that time (disallowing addresses above 2GB).

Reply:
Still, there is no reason to allow `-2` today.

Reply:
Hmm, what I worry about is forgetting to update this check at that time.

Actually also `-1 == 2^32 - 1` is a valid address with the eventual wasm64. So perhaps this should just check for `0`.